cluster/saltbase/salt/kube-dns/skydns-rc.yaml.sed

```diff
@@ -83,7 +83,7 @@ spec:
           name: dns-tcp-local
           protocol: TCP
       - name: dnsmasq
-        image: gcr.io/google_containers/dnsmasq:1.1
+        image: gcr.io/google_containers/dnsmasq-amd64:1.3
```

Comment:
Same thing applies here - the skydns-rc.yaml.sed file is machine generated from the .base file.